packages/app/src/systems/Home/components/HomeActions/HomeActions.tsx

```diff
@@ -12,11 +14,27 @@ export const HomeActions = ({
   receiveAction = () => {},
   sendAction = () => {},
 }: HomeActionsProps) => {
+  const { hasBalance, isLoading } = useAccount();
+
+  const sendButton = (
+    <Button
+      aria-label="Send Button"
+      onPress={sendAction}
+      isDisabled={isDisabled || !hasBalance}
+      css={styles.button}
+      isLoading={isLoading}
+    >
+      Send
+    </Button>
+  );
```

Comment:
because right after we need to check `hasBalance` to show it wrapped in a `Tooltip` or not, so to don't repeat the component entirely inside JSX, we can just save it inside a variable and do this:

```jsx
{hasBalance ? (
  sendButton
) : (
  <Tooltip content="You don't have balance to send">{sendButton}</Tooltip>
)}
```